services/rpcfilters/transaction_sent_to_upstream_event.go

```diff
@@ -4,23 +4,29 @@ import (
 	"errors"
 	"sync"
 
+	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/log"
-
-	"github.com/status-im/status-go/eth-node/types"
 )
 
+type PendingTxInfo struct {
+	Hash    common.Hash
+	Type    string
+	From    common.Address
```

Comment:
Not sure why the address matter for the transfer identity. Why not the usual ChainID + Hash?